app/src/main/res/menu/filesystem__menu.xml

```diff
@@ -12,6 +12,13 @@
     xmlns:tools="http://schemas.android.com/tools"
     tools:ignore="AlwaysShowAction">
 
+
+    <item
+        android:id="@+id/action_clear_selection"
+        android:icon="@drawable/ic_baseline_clear_24"
+        android:title="@string/clear_selection"
+        android:visible="false"
+        app:showAsAction="always" />
```

Comment:
@guanglinn here is the button I added. It's just a regular button in the toolbar